### PR TITLE
Overlays: describe default values for the animation object

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

**Review by Zedwag — APPROVED** [review]

File: api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md [hunk 1]
```diff
@@ -2,6 +2,109 @@
...
+                opacity: 0
+            }
+        }
+    }
+
 Use the [position](/api-reference/10%20UI%20Components/dxPopup/1%20Configuration/position.md '/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#position') property to specify the position in which the UI component is shown and from which it is hidden.
 
-Set the **animation** object to **null** or **undefined** to disable animation.
\ No newline at end of file
+Set the **animation** object to `null` or `undefined` to disable animation.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md]:
```suggestion
Set the **animation** object to `null` or `undefined` to disable animations.
```

File: api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md [hunk 2]
```diff
@@ -2,3 +2,46 @@
 id: dxToast.Options.animation
 type: Object
 ---
+The following code shows default values of the object depending on a device type:
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md]:
```suggestion
The following code shows default values of the object depending on the device type:
```

File: api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md [hunk 3]
```diff
@@ -2,3 +2,46 @@
...
+                position: {
+                    my: 'top',
+                    at: 'bottom',
+                    of: window
+                }
+            }
+        }
+    }
+
+Set the **animation** object to `null` or `undefined` to disable animation.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md]:
```suggestion
Set the **animation** object to `null` or `undefined` to disable animations.
```

It is **an** animation, **the** animation, or **animations**

File: api-reference/10 UI Components/dxOverlay/1 Configuration/animation/animation.md [hunk 4]
```diff
@@ -7,4 +7,4 @@ type: Object
 Configures UI component visibility animations. This object contains two fields: [show](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/animation/show.md '{basewidgetpath}/Configuration/animation/#show') and [hide](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/animation/hide.md '{basewidgetpath}/Configuration/animation/#hide').
 
 ---
-Set this object to **null** or **undefined** to disable animation.
\ No newline at end of file
+Set this object to `null` or `undefined` to disable animation.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxOverlay/1 Configuration/animation/animation.md]: an animation? the animation? animations in general?

File: api-reference/10 UI Components/dxPopover/1 Configuration/animation/animation.md [hunk 5]
```diff
@@ -4,7 +4,22 @@ type: Object
 ---
 ---
 ##### shortDescription
-Configures UI component visibility animations. This object contains two fields: [show](/api-reference/10%20UI%20Components/dxPopover/1%20Configuration/animation/show.md '/Documentation/ApiReference/UI_Components/dxPopover/Configuration/animation/#show') and [hide](/api-reference/10%20UI%20Components/dxPopover/1%20Configuration/animation/hide.md '/Documentation/ApiReference/UI_Components/dxPopover/Configuration/animation/#hide').
+Configures UI component visibility animations. This object contains two fields: [show]({basewidgetpath}/Configuration/animation/#show') and [hide]({basewidgetpath}/Configuration/animation/#hide).
 
 ---
-Set this object to **null** or **undefined** to disable animation.
\ No newline at end of file
+The following code specifies a default value of the object:
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxPopover/1 Configuration/animation/animation.md]: `a default value` The object can have different default values?

File: api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md [hunk 6]
```diff
@@ -2,6 +2,109 @@
 id: dxPopup.Options.animation
 type: Object
 ---
+The following code shows default values of the object depending on a device type:
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md]: Why `a device type`? Should it not be **the** device type?

**Review by dirk-pieterse — COMMENTED** [review]
Articles

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Components/dxOverlay/1 Configuration/animation/animation.md (hunk 4 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxOverlay/1 Configuration/animation/animation.md]:
```suggestion
Set this object to `null` or `undefined` to disable animations.
```

File: api-reference/10 UI Components/dxPopover/1 Configuration/animation/animation.md (hunk 5 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxPopover/1 Configuration/animation/animation.md]:
```suggestion
The following code specifies the default value of the object:
```

File: api-reference/10 UI Components/dxPopover/1 Configuration/animation/animation.md [hunk 7]
```diff
@@ -4,7 +4,22 @@ type: Object
...
+            to: 1
+        },
+        hide: {
+            type: 'fade',
+            from: 1,
+            to: 0
+        }
+    }
+
+Set this object to `null` or `undefined` to disable animation.
```
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxPopover/1 Configuration/animation/animation.md]:
```suggestion
Set this object to `null` or `undefined` to disable animations.
```

File: api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md (hunk 6 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md]:
```suggestion
The following code shows default values of the object depending on the device type:
```

File: api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md (hunk 1 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxPopup/1 Configuration/animation/animation.md]:
```suggestion
Set the **animation** object to `null` or `undefined` to disable animations.
```

File: api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md (hunk 2 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md]:
```suggestion
The following code shows default values of the object depending on the device type:
```

File: api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md (hunk 3 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxToast/1 Configuration/animation/animation.md]:
```suggestion
Set the **animation** object to `null` or `undefined` to disable animations.
```

**Pull request merged**